### ISSUE TITLE
'Keep on top' button appears last in the tab order

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1339]
**Describe the bug**
The 'Keep on top' button appears to be last in the tab order, resulting in focus order: NavigationView menu button > Calculator Display > _Everything else_ > Keep on top button.

**Steps To Reproduce**
1. Launch Narrator.
2. Launch Calculator app.
3. Navigate to 'Open Navigation' button and activate it.
4. Navigate to 'Standard' menu item and activate it.
5. 'Standard' calculator mode will open.
6. Set narrator scan mode using 'Caps+Space' keys.
7. Navigate to 'Open Navigation' and 'Keep on top' buttons using 'Up/Down arrow' keys in scan mode.
8. Observe the issue.

Note: Issue can be observed without Narrator or scan mode using keyboard focus.

**Expected behavior**
The 'Keep on top' button exists in tab order logically, resulting in focus order: NavigationView menu button > Calculator Display > Keep on top button > _Everything else_.

**Device and Application Information**
 - Application Version: 10.2005.23.0

**Requested Assignment**
I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by Callum-Shipton** [comment]
Hi,
I have had a look at this and it appears more complex than I had expected.

The Keep on top button is not contained within the NavigationView which contains the "Calculator Display" and "_Everything else_".
Thus, in it's current structure I believe it cannot alternate the tab order as you expect (Entering the NavView, exiting and re-entering).

Moving the button into the NavigationView to accommodate this would be a more involved change but possible.

Alternatively, if you would accept the tab ordering to instead be: NavigationView menu button > Keep on top button > Calculator Display > _Everything else_. The change would be trivial. Which looks pretty logical to me but would make the Calculator Display require an additional tab (which as the focus of the app, and is presumably your reasoning for expecting the tab order as you stated.)

Thoughts?

**Comment by grochocki** [comment]
> Alternatively, if you would accept the tab ordering to instead be: NavigationView menu button > Keep on top button > Calculator Display > Everything else.

Thanks for taking a look! I think this would be fine, especially since we land on the Calculator Display by default.
